app/lib/submission_builder/ty_2021/adv_ctc_irs1040.rb

```diff
@@ -42,33 +42,42 @@ def document
               dependent_xml(xml, dependent)
             end
             xml.ChldWhoLivedWithYouCnt qualifying_dependents.count { |qd| qd.qualifying_child?(2021) }
-            xml.OtherDependentsListedCnt qualifying_dependents.count { |qd| qd.qualifying_relative(2021) }
+            xml.OtherDependentsListedCnt qualifying_dependents.count { |qd| qd.qualifying_relative?(2021) }
             xml.TotalExemptionsCnt filer_exemption_count + qualifying_dependents.length
-            xml.TaxableInterestAmt 1 # 2b
-            xml.TotalIncomeAmt 1 # 9
-            xml.AdjustedGrossIncomeAmt 1 # 11
-            xml.TotalItemizedOrStandardDedAmt tax_return.standard_deduction # 12
+            xml.TotalItemizedOrStandardDedAmt tax_return.standard_deduction # line 12a in 2021v5.2/IndividualIncomeTax/Ind1040/IRS1040/IRS1040.xsd
+            xml.TotDedCharitableContriAmt tax_return.standard_deduction # 12c
+            xml.TotalDeductionsAmt tax_return.standard_deduction # 14
             xml.TaxableIncomeAmt 0 # 15
+
+            # Line 28: remaining amount of CTC they are claiming (as determined in flow and listed on 8812 14i
+            claimed_child_tax_credit = "0" # TODO: eventually replace this amount with the adv_ctc_amount_received when this is set in the RRC stories
+            xml.RefundableCTCOrACTCAmt claimed_child_tax_credit # 28
+            # maybe add this to be part of the stories for RRC
+
+            # Line 30: remaining amount of RRC they are claiming (as determined in flow). If they are not claiming anything, the Revenue Procedure is ambiguous as to whether this should be $0 or blank; but I recall last year having an issue with blank, so I would advocate $0. Rev Proc is clear IRS will not correct $0.
             xml.RecoveryRebateCreditAmt tax_return.claimed_recovery_rebate_credit # 30
-            xml.RefundableCreditsAmt tax_return.claimed_recovery_rebate_credit # 32
-            xml.TotalPaymentsAmt tax_return.claimed_recovery_rebate_credit # 33
-            xml.OverpaidAmt tax_return.claimed_recovery_rebate_credit # 34
-            xml.RefundAmt tax_return.claimed_recovery_rebate_credit # 35a
+
+            # Line 32, 33, 34, 35a: Line 28 + Line 30
+            total_payments = (claimed_child_tax_credit.to_i + tax_return.claimed_recovery_rebate_credit.to_i).to_s
```

Comment:
i think this should be eip3 amount instead of claimed_recovery_rebate_credit -- they can't get eip1 and 2 on thisreturn.

Reply:
I was using `claimed_recovery_rebate_credit` here because I think `outstanding_recovery_rebate_credit` (which is what determines claimed_recovery_rebate_credit) will get updated in a future RRC story to only make the calculations with EIP-3 [#181452309](https://www.pivotaltracker.com/story/show/181452309)